src/App.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import { Dimensions, SafeAreaView, StyleSheet, Text, View } from "react-native";
+import Playground from "./components/Playground";
+
+export default function App() {
+  const { width, height } = Dimensions.get("window");
+  const maxHeight = height * 0.23;
+  const [initialBallsPosition, setInitialBallPosition] = useState([
+    {
+      id: 1,
+      top: 0,
+      left: 0,
+    },
+  ]);
+  const moveBallToParticularPos = (
+    position: string,
+    playgroundBallId: number
+  ) => {
+    setInitialBallPosition((prev: any) => {
+      const updatedPositions = prev.map((item: any) => {
+        if (item.id === playgroundBallId) {
+          if (position === "left") {
+            return { ...item, left: Math.max(0, item.left - 10) };
+          }
+          if (position === "right") {
+            return { ...item, left: Math.min(width - 40, item.left + 10) };
```

Comment:
What is 40? Please make it such that the value is dynamic from the provided size of the dot.